hexapod_dart/src/hexapod_dart_simu.cpp

```diff
@@ -334,3 +291,86 @@ void HexapodDARTSimu::_add_floor()
 
     _world->addSkeleton(floor);
 }
+
+double HexapodDARTSimu::_min_dist_angle(double a1, double a2)
+{
+    // compute minimum signed distance between 2 angles
+    double res = a1 - a2;
+    // res = (res + 180) % 360 - 180;
+    // mod = (a, n) -> a - floor(a/n) * n
+    res = res + 180.0;
+    res = res - std::floor(res / 360.0) * 360.0;
+    res = res - 180.0;
+
+    // we want [-180,180] range
+    while (res < -180)
+        res += 360;
+    while (res > 180)
+        res -= 360;
+    return res;
+}
+
+void HexapodDARTSimu::_check_duty_cycle()
+{
+    auto rob = this->robot();
+
+    for (unsigned i = 0; i < 6; ++i) {
+        std::string leg_name = "leg_" + std::to_string(i) + "_3";
+        dart::dynamics::BodyNodePtr tmp;
```

Comment:
What makes this variable worth of the status "temporary" ? I think it is too high a distinction for it.